rootfs/standard/var/www/mynode/static/js/manage_apps.js

```diff
@@ -127,4 +127,53 @@ function toggleEnabled(short_name, full_name, enable, return_page="") {
                                 window.location.href="/toggle-enabled?app="+short_name+r
                            });
     }
+}
+
+// ==========================================
+// Manage app storage (data_folder)
+// ==========================================
+
+function backup_data_folder_via_api(name, short_name) {
+    if ( confirm("Are you sure you want to backup "+name+"? This will stop, backup data and start app.") ) {
+        $('#loading_spinner_message').html("Making backup...");
+        $('#loading_spinner_overlay').fadeIn();
+        $.get('/api/backup_data_folder?short_name=' + short_name)
```

Comment:
Keep the name "app" at the argument, it's more consistent with other APIs. However, still put short_name into the string, like:

$.get('/api/backup_data_folder?app=' + short_name)

Reply:
I see. I'll change it this way.

Reply:
> I see. I'll change it this way.

still wrk in progress